docs/dashboard/configuration/shipping.md

```diff
@@ -53,17 +53,17 @@ These two methods cover most of the basic use cases, for example:
 * Increasing the shipping price for heavy orders.
 
 
-### ZIP Codes
+### Postal codes
 
-This section allows you to exclude shipping method for customers with certain ZIP codes.
-If customer shipping address ZIP Code is within any of specified ranges, that shipping method will be unavailable for that customer.
+This section allows you to exclude shipping method for customers with certain postal codes.
```

Comment:
This section allows you to exclude a shipping method for customers in certain postal codes.